syntax="proto2";

package aasdk.proto.ids;

message PhoneStatusChannelMessage
{
  enum Enum
  {
    NONE = 0x0000;
    PHONE_STATUS = 0x8001;
    PHONE_STATUS_INPUT = 0x8002;
  }
}
option optimize_for = SPEED;
